blueprints/devops/jfrog-cli/ops2deb.yml

```diff
@@ -0,0 +1,17 @@
+name: jfrog-cli
+matrix:
+  architectures:
+    - amd64
+  versions:
+    - 2.59.1
+homepage: https://docs.jfrog-applications.jfrog.io/jfrog-applications/jfrog-cli
+summary: JFrog CLI is a client that provides a simple interface that automates access
```

Comment:
No need to repeat the tool name: `client that provides a simple interface that automates access to JFrog products`